src/gui/accelerators.c

```diff
@@ -185,7 +185,8 @@ void dt_accel_register_lib_for_views(dt_lib_module_t *self, dt_view_type_flags_t
   accel->local = FALSE;
   // we get the views in which the lib will be displayed
   accel->views = views;
-  darktable.control->accelerator_list = g_slist_prepend(darktable.control->accelerator_list, accel);
+  if (!dt_accel_find_by_path(accel->path))
+    darktable.control->accelerator_list = g_slist_prepend(darktable.control->accelerator_list, accel);
```

Comment:
we need to free accel (or better make the test before the allocation)

Reply:
Checking before allocation as you suggested.
the function is called multiple times, not only when initializing accels, but also when switching views (which make the list grow continuously) and when registering lua modules.
So perhaps there is a better way to fix at higher level, but I don't want to break other things